.obs/freeciv21.spec

```diff
@@ -0,0 +1,121 @@
+#
+# RPM spec file for freeciv21 to run on Open Build Service (OBS)
+# Project Home: https://build.opensuse.org/project/show/home:longturn
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: Louis Moureaux <m_louis30@yahoo.com>
+# SPDX-FileCopyrightText: James Robertson <jwrober@gmail.com>
+#
+Name:           freeciv21
+Version:        3.2
+Release:        %autorelease -p pre -s %{__scm_source_timestamp}
+License:        GPL-3.0
+Group:          Amusements/Games/Strategy/Turn Based
+Summary:        Develop Your Civilization from Humble Roots to a Global Empire
```

Comment:
We may want to take [these guidelines](https://en.opensuse.org/openSUSE:Package_description_guidelines) into account (but it's not an official package so a neutral tone isn't needed.

Reply:
I personally think our global slogan is fine here. I guess we could just go with "Freeciv21 4x Civ Game"

Reply:
I've done this for the LT account:

> We develop and play Freeciv21. Develop your civilization from humble roots to a global empire!

Maybe:

> The Freeciv21 4x game. Develop your civilization from humble roots to a global empire!